package.json

```diff
@@ -22,60 +22,74 @@
     "@alrra/travis-scripts": "3.0.1",
     "@google-cloud/storage": "1.6.0",
     "ajv": "5",
-    "babel-core": "6.25.0",
-    "babel-eslint": "8.1.2",
-    "babel-loader": "7.1.1",
+    "babel-core": "6.26.3",
+    "babel-eslint": "8.2.3",
+    "babel-loader": "7.1.4",
     "babel-plugin-react-intl": "2.3.1",
     "babel-plugin-transform-class-properties": "6.24.1",
     "babel-plugin-transform-flow-strip-types": "6.22.0",
-    "babel-preset-env": "1.6.0",
+    "babel-preset-env": "1.6.1",
     "babel-preset-flow": "6.23.0",
     "babel-preset-react": "6.24.1",
     "babel-preset-stage-2": "6.24.1",
-    "chalk": "2.3.0",
+    "chalk": "2.4.1",
     "chromeless": "1.4.0",
     "convict": "4.0.1",
     "coveralls": "2.13.1",
-    "css-loader": "^0.28.4",
-    "eslint": "4.14.0",
+    "css-loader": "0.28.11",
+    "eslint": "4.19.1",
     "eslint-config-prettier": "2.9.0",
     "eslint-plugin-flowtype": "2.40.1",
     "eslint-plugin-react": "7.5.1",
+    "file-loader": "1.1.11",
     "gettext-parser": "1.2.2",
     "github": "12.0.1",
     "glob": "7.1.2",
     "googleapis": "23.0.0",
     "gunzip-maybe": "1.4.1",
+    "isomorphic-style-loader": "4.0.0",
     "mkdirp": "0.5.1",
     "node-insights": "1.0.16",
-    "prettier": "1.5.2",
+    "node-sass": "4.9.0",
+    "postcss-css-variables": "0.8.1",
+    "postcss-cssnext": "3.1.0",
+    "postcss-loader": "2.1.4",
+    "prettier": "1.12.1",
     "raw-loader": "0.5.1",
     "react-dev-utils": "3.0.2",
-    "style-loader": "0.18.2",
-    "sw-precache-webpack-plugin": "0.11.4",
+    "react-hot-loader": "4.1.2",
+    "sass-loader": "7.0.1",
+    "style-loader": "0.21.0",
+    "sw-precache-webpack-plugin": "0.11.5",
+    "swarm-constants": "1.2.99",
```

Comment:
probably worth making this a peer dep?

Reply:
leaving as `dependency` only for now. Probably can be moved to `peerDep` in a follow up.